src/core/database/query.helpers.ts

```diff
@@ -997,6 +997,12 @@ export function listReturnBlock<T = any>(
   isSecuredSort: boolean,
   sort?: string | ((sortStr: string) => string)
 ) {
+  const securedSortMap: Partial<Record<typeof sortInput, string>> = {
+    name: `lower(node.${sortInput}.value)`,
+  };
+  const insecuredSortMap: Partial<Record<typeof sortInput, string>> = {
+    name: `lower(node.${sortInput})`,
+  };
```

Comment:
This is a generic function and should not be making guesses about properties.

This function is also deprecated. I didn't realize it was still in use by orgs. I'll create an issue to update it.